doc/source/templates/04_finetuning_llms_with_deepspeed/finetune_hf_llm.py

```diff
@@ -399,7 +399,7 @@ def training_function(kwargs: dict):
                 s_fwd = time.time()
                 outputs = model(**batch)
                 loss = outputs.loss
-                loss_sum += loss
```

Comment:
@woshiyyya I guess this and `loss.detach()` should end up doing the same thing?

Reply:
I was going for loss.detach() and @woshiyyya told me that he used .item() when doing this.
I have since spun up. a workspace to validate @woshiyyya 's findings.

I could run a multiple-epoch training without signs of a leak with the commit here on V100s.
Without, it was OOming.

CC @kouroshHakha

Reply:
Yes they should have the same effect. I saw the GRAM usage are constant during training.